database/things/css.pldb

```diff
@@ -3,6 +3,7 @@ appeared 1996
 type stylesheetLanguage
 creators Håkon Wium Lie
 webRepl https://playcode.io/css/
+documentation https://devdocs.io/css/
```

Comment:
wow devdocs is a pretty cool site. not sure if ive seen it before. nice find!

Reply:
Yes @breck7 , it is a pretty cool site.